src/settings-ui/Settings.UI.Library/Utilities/Helper.cs

```diff
@@ -135,5 +134,10 @@ public static int CompareVersions(string version1, string version2)
         }
 
         public const uint VirtualKeyWindows = interop.Constants.VK_WIN_BOTH;
+
+        public static bool Windows11()
+        {
+            return Environment.OSVersion.Version.Major >= 10 && Environment.OSVersion.Version.Build >= 22000;
```

Comment:
I'm just searching and I see messages like https://developercommunity.visualstudio.com/t/environmentosversionversion-return-62-for-windows/670661
Is that related or not? And shouldn't it be `>= 11` or does the difference (between 10 and 11) depend more on the build version?
On my desktop, Win.10, 21H2 under `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion`, values "Currentbuild" == "19044", "CurrentVersion" == "6.3" and "UBR" == "1586".

Reply:
@jay-o-way
Win11 is in the first value 10 too. It's 10.0.*!

Reply:
Windows 11 is 10.0.x with first build 22000.
I have tested the code on Windows 10 and Windows 11 and `Environment.OSVersion` is returning the correct value.